VirtoCommerce.Platform.Modules/Bundling/FileCollector.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Generic;
+using System.IO;
+using VirtoCommerce.Platform.Core.VersionProvider;
+
+namespace VirtoCommerce.Platform.Modules.Bundling
+{
+    public abstract class FileCollector : ICollector
```

Comment:
use name BundleProvider and move it to middleware for collecting ModuleFile[] one time
because each time (login/refresh) collect array of files